rust/arrow/src/util/pretty.rs

```diff
@@ -80,8 +84,8 @@ macro_rules! make_string {
     }};
 }
 
-/// Get the value at the given row in an array as a string
-fn array_value_to_string(column: array::ArrayRef, row: usize) -> Result<String> {
+/// Get the value at the given row in an array as a String
+pub fn array_value_to_string(column: &array::ArrayRef, row: usize) -> Result<String> {
```

Comment:
I made this `pub` so I could call it from datafusion/src/test/sql.rs which currently has a copy/pasted version of pretty printing in `array_str`: https://github.com/apache/arrow/blob/master/rust/datafusion/tests/sql.rs#L646

Reply:
See https://github.com/apache/arrow/pull/8333